src/style.css

```
@@ -0,0 +1,126 @@
+.body {
+  margin: 0;
+  --color: #00acdc;
+  --star_size: 16px;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card_img {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 40px;
+}
+
+.card_text {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin: 0 16px 4px;
+}
+
+.card_product_code {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin: 0 16px 16px;
+}
+
+.stars_reviews {
+  display: flex;
+  margin: 0 16px 24px;
```

instead of setting horizontal margins for each part of card set paddings for .card
check other elements.
```suggestion
  margin-bottom: 24px;
```